v1/authorization/storage_authorization.go

```diff
@@ -68,6 +69,17 @@ func (s *Store) CreateAuthorization(ctx context.Context, tx kv.Tx, a *influxdb.A
 		a.ID = id
 	}
 
+	ts := tenant.NewStore(s.kvStore)
```

Comment:
I went back and forth about DI here, I ultimately went this way because testing ended up not being too hard. Happy to consider it though if there are stronger opinions.